sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_devices_last_seen_v1/query.sql

```diff
@@ -0,0 +1,85 @@
+WITH _current AS (
+  SELECT
+    `timestamp` AS last_seen_date,
```

Comment:
In most of our tables we usually use a submission_date but we are going with a last_seen_date here. Is this a common pattern with all FxA data or are we making a change here? If in other FxA data it's last seen date then no concerns. If we are making a change then we should have a discussion.

Reply:
that's fair, I think I did that because it's was slightly different way of getting this value. In other workflows it was just setting the value to Airflow's execution date whereas I was actually using the value from the existing record. I changed the naming to keep it consistent across.